Main.java

```diff
@@ -37,47 +39,61 @@ public static void main(String[] args) {
 					for (int j = i; j <= Birthdate.length(); j++) {
 						if (j >= i) {
 							String password = Parameter_Value.get(0) + Birthdate.substring(i, j);
-							fw.write(password);
-							fw.write("\n");
+							if (IsUnique(Old_Passwords, password)) {
+								fw.write(password);
+								fw.write("\n");
+							}
 							for (int len = 0; len < Parameter_Value.size(); len++) {
 								String name = Parameter_Value.get(len);
 								for (int s = 0; s < Special_Char.length; s++) {
 									password = name + Special_Char[s] + Birthdate.substring(i, j);
-									fw.write(password);
-									fw.write("\n");
+									if (IsUnique(Old_Passwords, password)) {
+										fw.write(password);
+										fw.write("\n");
+									}
 
 									if (name != name.toLowerCase()) {
 										password = name.toLowerCase() + Special_Char[s] + Birthdate.substring(i, j);
-										fw.write(password);
-										fw.write("\n");
+										if (IsUnique(Old_Passwords, password)) {
+											fw.write(password);
+											fw.write("\n");
+										}
 									}
 
 									if (name != name.toLowerCase()) {
 										password = name.toUpperCase() + Special_Char[s] + Birthdate.substring(i, j);
-										fw.write(password);
-										fw.write("\n");
+										if (IsUnique(Old_Passwords, password)) {
+											fw.write(password);
+											fw.write("\n");
+										}
 									}
 								}
 
 								for (int s1 = 0; s1 < Special_Char.length; s1++) {
 									for (int s2 = 0; s2 < Special_Char.length; s2++)
 										password = name + Special_Char[s1] + Birthdate.substring(i, j)
 												+ Special_Char[s1];
```

Comment:
Special chars are coming same in output due to this small typo.

password = name + Special_Char[s1] + Birthdate.substring(i,j) + Special_Char[s2];